projects/packages/sync/src/modules/class-woocommerce-hpos-orders.php

```diff
@@ -302,6 +307,34 @@ private function filter_order_data( $order_object ) {
 		return $filtered_order_data;
 	}
 
+	/**
+	 * Returns all possible order status keys using 'wc_get_order_statuses', if possible..
+	 *
+	 * @access protected
+	 *
+	 * @return array Filtered order metadata.
+	 */
+	protected function get_all_possible_order_status_keys() {
+		$order_statuses    = array( 'checkout-draft', 'auto-draft', 'trash' );
+		$wc_order_statuses = array();
+		if ( function_exists( 'wc_get_order_statuses' ) ) {
```

Comment:
When is `wc_get_order_statuses` not available

Reply:
When WooCommerce is not active I guess, but since it's an external function I think it's safer to check if it exists first before attempting to use it.